src/routes/display/+layout.svelte

```diff
@@ -1,10 +1,29 @@
 <script lang="ts">
   import QR from "$lib/assets/qr-code.svg";
+  import { pb } from "$lib/stores/authStore";
+  import { onMount } from "svelte";
+
+  export let data: { screenMessageRecord };
+  let screenMessage = data.screenMessageRecord[0];
```

Comment:
I fremtiden bør nok dette erstattes med noe som finner den ene record fra collection som er satt til visible, men akkurat nå er dette bra, det bør gjøre i annen pr, samtidig som man implementerer flere valg for de ansatte.